srv/salt/ceph/monitoring/grafana/init.sls

```diff
@@ -23,6 +25,17 @@ grafana-dashboards-ceph:
         enabled = true
         path = /var/lib/grafana-dashboards-ceph
 
+wait-for-grafana-http:
+  cmd.run:
+    - require:
+      - service: grafana-server
+    - name: |
+         i=0
+         until curl -s -H "Content-Type: application/json" -XGET http://admin:admin@localhost:3000/api/datasources || [ $i = 24 ]; do
+           sleep 5
+           i=$((i + 1))
+         done;
+
```

Comment:
If we decide that we want to display an error message, we might need to re-design the loop, because this one might succeed on the 24th try and then wrongly print the error message.